setup.py

```diff
@@ -17,7 +18,7 @@ def get_version(*file_paths):
     Extract the version string from the file at the given relative path fragments.
     """
     filename = os.path.join(os.path.dirname(__file__), *file_paths)
-    version_file = open(filename).read()
+    version_file = open_as_of_py3(filename, encoding='utf-8').read()
```

Comment:
why not just `io.open`?

Reply:
Because importing specifically what you need rather than a whole module has been drilled into me by the javascript gods. Even if it doesn't facilitate any optimizations it still feels more explicit to me this way.